obs/obs-service-api/src/main/java/com/adorsys/webank/obs/dto/response/ErrorResponse.java

```diff
@@ -0,0 +1,39 @@
+package com.adorsys.webank.obs.dto.response;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Schema(description = "Response object for API errors")
+public class ErrorResponse {
+
+    @Schema(description = "HTTP status code", example = "400")
+    private int status;
+
+    @Schema(description = "Error code for specific error type", example = "INVALID_INPUT")
+    private String errorCode;
```

Comment:
Missing something here: 
```suggestion
private ErrorCode errorCode;
```